### DISCUSSION TITLE
geom object spawn under the mesh object

### DISCUSSION THREAD

**Discussion by takeru1205** [google-deepmind/mujoco#22] [Asking for Help] [answered]
Hi there,

I'm trying to replace the table which is used in OpenAI gym FetchSlide task.
The table is created a stl file. So I loaded it as a mesh file.
I just changed geom object to mesh object. Although, the pack object is placed under the table.

Are there ways to solve that?

I changed xml file like below.
https://github.com/openai/gym/blob/3eb699228024f600e1e5e98218b15381f065abff/gym/envs/robotics/assets/fetch/slide.xml#L18-L20
```
<body name="table0" pos="0.75 0.2 0.">
    <geom name="table0" type="mesh" mesh="table0:table2_wide_108" material="table_mat" pos="0 0 0" fitscale="2" friction="0.1 0.005 0.0001"></geom>
</body>
```

my stl file: https://drive.google.com/file/d/1KCZFF5Rk-ZPT8H7xYQJHPX18LdH19EUd/view?usp=sharing

Thanks,

**Comment by yuvaltassa** [comment, marked as answer]
Hi, 
Please read [this section](https://mujoco.readthedocs.io/en/latest/XMLreference.html#asset-mesh) in the documentation to understand how MuJoCo handles mesh positioning.
